addons/crm/models/crm_team.py

```diff
@@ -382,8 +382,7 @@ def _allocate_leads(self, work_days=1):
         Heuristic of this method is the following:
           * find unassigned leads for each team, aka leads being
             * without team, without user -> not assigned;
-            * not in a won stage, and not having False/0 (lost) or 100 (won)
-              probability) -> live leads;
+            * not won nor lost -> live leads;
```

Comment:
Note that the domain does not seem to match this, as it only excludes won?

I guess we rather ignore inactive leads, which also makes sense, but is not exactly the same.